src/account/FiatExchange.tsx

```diff
@@ -23,9 +23,20 @@ import { navigateToURI } from 'src/utils/linking'
 import Logger from 'src/utils/Logger'
 
 function FiatExchange() {
+  return <FiatExchangeSection />
+}
+
+export function FiatExchangeSection({
+  hideAddFunds,
+  hideDrawerTopNav,
```

Comment:
nit: should we use `show` instead of `hide`, Since we've been using `show` for our parameters?

Reply:
➕